src/shared/components/community/create-community.tsx

```diff
@@ -68,6 +68,11 @@ export class CreateCommunity extends Component<
     const res = await HttpService.client.createCommunity(form);
 
     if (res.state === "success") {
+      const myUser = UserService.Instance.myUserInfo!;
+      UserService.Instance.myUserInfo?.moderates.push({
+        community: res.data.community_view.community,
+        moderator: myUser.local_user_view.person,
+      });
```

Comment:
This is a kludge, but there isn't a better way to do it in this codebase at the moment.